nextflow_schema.json

```diff
@@ -9,374 +9,419 @@
             "title": "Input/output options",
             "type": "object",
             "fa_icon": "fas fa-terminal",
-            "description": "Define where the pipeline should find input data and write output data.",
-            "required": [
-                "input",
-                "input_type"
-            ],
+            "description": "Define where the pipeline should find input data and write results.",
+            "required": ["input", "input_type", "outdir"],
             "properties": {
                 "input": {
                     "type": "string",
-                    "fa_icon": "fas fa-dna",
+                    "fa_icon": "fas fa-copy",
```

Comment:
Yeah would be great to have mimetype added here as mentioned in my first review: See an example in viralrecon. 

https://github.com/nf-core/viralrecon/blob/920466a3e4acf540e92950e6d221c20a000a8662/nextflow_schema.json#L18